code/modules/clothing/under/miscellaneous.dm

```diff
@@ -599,6 +612,12 @@
 	worn_state = "pcrcsuit"
 	gender_icons = 1
 
+/obj/item/clothing/under/solgov/utilityskirt
+	name = "Utility Skirt"
+	desc = "A comfortable turtleneck and black utility skirt."
+	icon_state = "contractor_skirt"
+	worn_state = "contractor_skirt"
```

Comment:
Doesn't work because it's searching in another file, not the one you added the icon to.

`UNIT_TEST ##: /datum/gear/uniform/contractor_skirt - /obj/item/clothing/under/solgov/utilityskirt: Did not find the icon state 'contractor_skirt' in the icon 'modular_boh/icon/obj/obj_under_solgov.dmi'.`

`UNIT_TEST ##: !!! FAILURE !!! [LOADOUT: Entries shall have valid icon states]: One or more /datum/gear definitions had paths with invalid icon states.`